cargo-bolero/src/rmc.rs

```diff
@@ -0,0 +1,27 @@
+use crate::{exec, test, Selection};
+use anyhow::{anyhow, Result};
+use bit_set::BitSet;
+use core::cmp::Ordering;
+use std::{
+    fs,
+    io::{BufRead, BufReader, Result as IOResult, Write},
+    path::{Path, PathBuf},
+    process::Command,
+};
```

Comment:
How many of these dependencies do we actually need here?

Reply:
Currently, we don't need most of them. I'll take them out, and once we start supporting/processing the arguments passed to `test`, we can add the needed ones back.